azext_edge/tests/edge/support/create_bundle_int/test_auto_int.py

```diff
@@ -69,6 +70,9 @@ def test_create_bundle(cluster_connection, ops_service, bundle_dir, mq_traces, t
     arc_namespace = namespaces.get("arc")
     certmanager_namespace = namespaces.get("certmanager")
 
+    # cleanup paths in walk_result that are not in namespaces (user customized namespaces)
```

Comment:
Does this mean the support bundle is capturing information that it shouldn't be?

Reply:
following up with service team